test/cctest/test_util.cc

```diff
@@ -90,10 +90,6 @@ TEST(UtilTest, ToLower) {
   EXPECT_EQ('a', ToLower('A'));
 }
 
-namespace node {
-  void LowMemoryNotification() {}
-}
-
```

Comment:
Intentional change?  (I'm guessing yes, conflict with the function from util.cc?)